script.js

```diff
@@ -0,0 +1,44 @@
+let saladMenuFileData = [], prices = [];
+let bill = 0, divContent="<table>";
+
+fetch("/salads.json")
+  .then((response) => response.json())
+  .then((saladMenuFileData) => createMenu(saladMenuFileData));
+
+  function setPrice(counter,saladMenuFileData){
+    if((counter + 1) % 3 == 0) return 15;

```

Comment:
Good modulo operator. O to chodziło. Jeżeli chodzi o block scope (czyli wszystkie loopy i conditionale w języku o zakresie blokowym klamerek, if, else, switch, for, while itd.) to należy dawać klamerki dla lepszego czytania kodu.